lib/trento/application/projectors/database_projector.ex

```diff
@@ -159,6 +160,25 @@ defmodule Trento.DatabaseProjector do
     end
   )
 
+  project(
+    %DatabaseInstanceDeregistered{
+      instance_number: instance_number,
+      host_id: host_id,
+      sap_system_id: sap_system_id,
+      deregistered_at: _deregistered_at
```

Comment:
If you don't care about `deregistered_at`, do you even need to pattern match on it here?